cmd/main.go

```diff
@@ -34,6 +36,7 @@ func main() {
 
 	statusCryptoTransferRepository := status.NewStatusRepository(db, "CRYPTO_TRANSFER")
 	statusConsensusMessageRepository := status.NewStatusRepository(db, "HCS_TOPIC")
```

Comment:
We can move those 3 lines after the `transactionRepository` just so that we can have the initialisation of repositories together